src/dscom/RegistrationServices.cs

```diff
@@ -283,6 +283,18 @@ private static IReadOnlyCollection<Type> GetComRegistratableTypes(Assembly assem
     {
         static bool TypeMustBeRegistered(Type type)
         {
+            // Only public types are com visible
+            if (!type.IsPublic && !type.IsNestedPublic && !type.IsByRef)
+            {
+                return false;
+            }
```

Comment:
Any issues with that? This filter is only applied for assembly.GetExportedTypes() so that this should never be hit...

Reply:
yes, it looks like that is the case. Shame on me, because i used my filtering for com visible types i used in an automigration project.

this change is obsolet.